# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../vnuml/vnuml-1.8.9-no-tun-device-check.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- vnuml-1.8.9/configure.ac.orig	2011-05-20 12:09:34.500002359 +0200
+++ vnuml-1.8.9/configure.ac	2011-05-20 12:10:32.512002358 +0200
@@ -247,14 +247,6 @@
 AC_SUBST(PERL_MODULES_INSTALLROOT)
 AC_CHECK_FILE([$PERL_MODULES_INSTALLROOT], AC_SUBST(PERL_MODULES_INSTALLROOT), AC_MSG_ERROR([Invalid Perl modules installroot directory: $VNUML_PERL_MODULES_INSTALLROOT. Please set the appropriate installroot directory using the --with-perl-modules-installroot.]))
 
-dnl tun device used for cloning
-AC_ARG_WITH(tun,
-	    AC_HELP_STRING([--with-tun=DEV],
-	    		   [tun device used for cloning (default: /dev/net/tun)]),
-            VNUML_TUN=$withval,
-            VNUML_TUN=/dev/net/tun)
-AC_CHECK_FILE([$VNUML_TUN], AC_SUBST(VNUML_TUN), AC_MSG_ERROR([tun device $VNUML_TUN not found.  Please set the appropriate device using the --with-tun option.]))
-
 AC_ARG_WITH(tun_group,
 	    AC_HELP_STRING([--with-tun_group=GROUP],
 	    		   [group to own tun device (default: uml-net)]),
